File: widgetssdk/src/main/java/com/glia/widgets/notification/domain/RemoveScreenSharingNotificationUseCase.java
package com.glia.widgets.notification.domain;

import com.glia.widgets.notification.device.INotificationManager;

public class RemoveScreenSharingNotificationUseCase {
    private final INotificationManager notificationManager;

    public RemoveScreenSharingNotificationUseCase(INotificationManager notificationManager) {
        this.notificationManager = notificationManager;
    }

    public void execute() {
        this.notificationManager.removeScreenSharingNotification();
    }
}

File: widgetssdk/src/main/java/com/glia/widgets/internal/notification/domain/RemoveScreenSharingNotificationUseCase.kt
package com.glia.widgets.internal.notification.domain

import com.glia.widgets.internal.notification.device.INotificationManager

internal class RemoveScreenSharingNotificationUseCase(private val notificationManager: INotificationManager) {
    operator fun invoke() = notificationManager.removeScreenSharingNotification()
}
